assets/js/blocks/cart-checkout/checkout/block.js

```diff
@@ -158,6 +159,19 @@ const Checkout = ( { attributes, scrollToTop } ) => {
 		return <CheckoutOrderError />;
 	}
 
+	const loginPrompt = () =>
+		! CHECKOUT_SHOW_LOGIN_REMINDER ? null : (
```

Comment:
Nitpicking but what do you think about replacing the ternary with an AND operator? It should work as well and I think it simplifies the code a bit.
```suggestion
		CHECKOUT_SHOW_LOGIN_REMINDER && (
```

Reply:
+1